src/probnum/utils/arrayutils.py

```diff
@@ -47,10 +45,9 @@ def atleast_1d(*rvs):
 def as_colvec(
     vec: Union[np.ndarray, "probnum.RandomVariable"]
 ) -> Union[np.ndarray, "probnum.RandomVariable"]:
-    """Transform the given vector or random variable to column format.
-
-    Given a vector (or random variable) of dimension (n,) return an array with
-    dimensions (n, 1) instead. Higher-dimensional arrays are not changed.
+    """Transform the given vector or random variable to column format. Given a vector
+    (or random variable) of dimension (n,) return an array with dimensions (n, 1)
+    instead. Higher-dimensional arrays are not changed.
```

Comment:
isn't this the kind of docstring style that pylint and docformatter dislike? If not, no complaints, I am just a bit confused.

Reply:
I am not sure I understand. I think this was just an automatic reformatting by `docformatter`. What part of it would pylint or docformatter "dislike"?

Reply:
> What part of it would pylint or docformatter "dislike"?
I thought docformatter makes a linebreak after the first sentence (here after "format"). Though I just mention this for clarity of my comment, not because anything needs to be done here.